src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -738,7 +738,22 @@ private Packaging.Core.PackageDependency CreateDependencyFromProject(dynamic pro
         private void AddOutputFiles(Packaging.PackageBuilder builder)
         {
             // Get the target framework of the project
-            FrameworkName targetFramework = TargetFramework;
+            NuGetFramework nugetFramework;
+            if (_usingJsonFile && builder.TargetFrameworks.Any())
+            {
+                if (builder.TargetFrameworks.Count > 1)
+                {
+                    var message = string.Format(
+                    CultureInfo.InvariantCulture,
+                    LocalizedResourceManager.GetString("Error_MultipleTargetFrameworks"));
+                    throw new CommandLineException(message);
+                }
+                nugetFramework = builder.TargetFrameworks.First();
```

Comment:
Is it possible to have zero if the project.json has no frameworks section, or is this populated by the project file in that case?